### DISCUSSION TITLE
直播间颜文字按钮恢复问题、视频弹窗问题

### DISCUSSION THREAD

**Discussion by DIXX-qqfr** [the1812/Bilibili-Evolved#2135] [功能建议 / Ideas]
B站把直播间颜文字的按钮删掉了，虽然我自己也存了那些，但还是希望把按钮恢复更方便；
视频现新增了一种打星的弹窗，没有被“删除投票弹窗”功能屏蔽，希望一起屏蔽

**Comment by the1812** [comment]
投票弹窗屏蔽会支持
恢复被删掉的功能可能得去 https://github.com/MotooriKashin/Bilibili-Old 问问
